pyomo/contrib/trustregion/plugins.py

```diff
@@ -15,9 +15,9 @@
 from pyomo.contrib.trustregion.readgjh import readgjh
 
 logger = logging.getLogger('pyomo.contrib.trustregion')
-fh = logging.FileHandler('debug_vars.log')
-logger.setLevel(logging.DEBUG)
-logger.addHandler(fh)
+#fh = logging.FileHandler('debug_vars.log')
+#logger.setLevel(logging.DEBUG)
+#logger.addHandler(fh)
```

Comment:
Can these commented lines be removed?

Reply:
Yes, although I was going to leave removing them to @dmvigi - as she may want them there as reference for use in future development of this solver.

Reply:
That's fine, just wanted to check.